Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM ubuntu
+FROM node:8-alpine
 
-RUN apt-get update
+# Templating from registration repo
+RUN apk update && apk add bash
+RUN apk add git
+RUN npm install npm@"~5.4.0" && rm -rf /usr/local/lib/node_modules && mv node_modules /usr/local/lib
```

Comment:
@petschekr why did we do this again?

Reply:
npm's maintainers were introducing breaking changes in minor versions. This locks the version to 5.4.x. Probably not necessary anymore now that npm v5 has settled down.